doc/dev/developer_guide/dash-devel.rst

```diff
@@ -2267,6 +2267,46 @@ Usage example:
   }
 
 
+How to create an issue in the Ceph Issue Tracker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Ceph-Dashboard provides two ways to create an issue in the Ceph Issue Tracker, 
+either using the Ceph command line interface or by using the Ceph Dashboard
+user interface. 
+
+To create an issue in the Ceph Issue Tracker, a user needs to have an account
+on the issue tracker. Under the ``my account`` tab in the Ceph Issue Tracker,
+the user can see their API access key. This key is used for authentication
+when creating a new issue. To store the Ceph API access key, in the CLI run:
+
+``ceph dashboard set-ceph-tracker-api-key -i <file-containing-key>``
+
+Then on successful update, you can create an issue using:
+
+``ceph dashboard create issue <project> <tracker> <subject> <description>``
```

Comment:
This `<tracker>` is confusing me. It should be `<tracker_type>` IMO.

Reply:
How about `<bug/feature>` instead of `<tracker_type>`

Reply:
I'll go with generalising and describing what all tracker types we can add. What if we decide to include `cleanup` in here? We have to call it `<bug|feature|cleanup>`. `<tracker_type>` or something similar will be good IMHO.

Reply:
Yeah, that makes more sense. Thanks

Reply:
What about using the official Redmine naming schema? This is already used in the [Feedback class](https://github.com/ceph/ceph/pull/42090/files#diff-c0432f9c9eaa294660d9458247c3eea7710457d107bb9b55305fc73a33644036R7).
```suggestion
``ceph dashboard create issue <project> <tracker_id> <subject> <description>``
```
Reference: https://www.redmine.org/projects/redmine/wiki/Rest_Trackers

Reply:
@votdev We found that tracker_id could be misleading at first sight as you think of it as the tracker id number.
Tracker_id in reality means tracker type id. This can be re-visited again in the follow-up PR.